### PR TITLE
feat: added migration for old app colors to new colors

### LINKED ISSUE #7609: Migrate old colors
We need to migrate the background colors of the apps to new colors. Here is the mapping:

https://www.figma.com/file/i80YbQLLs5mXthjIHclPpX/Appsmith-Design-System?node-id=13%3A108

### PR DESCRIPTION
**Pull request by nayan-rafiq**
## Description
Updates old app background colors to new colors.

Fixes #7609

## Type of change
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
- Tested on local machine

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/server/appsmith-server/src/main/java/com/appsmith/server/migrations/DatabaseChangelog.java [hunk 1]
```diff
@@ -3240,4 +3240,32 @@ public void migrateMongoPluginDynamicBindingListUqi(MongoTemplate mongoTemplate)
             mongoTemplate.save(mongoAction);
         }
     }
+
+    @ChangeSet(order = "089", id = "migrate-old-app-color-to-new-colors", author = "")
+    public void migrateOldAppColorsToNewColors(MongoTemplate mongoTemplate) {
```
Comment (abhvsn) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/migrations/DatabaseChangelog.java:3245]: Can we use `MongockTemplate` as `MongoTemplate` is deprecated AFAIK

**Review by abhvsn — CHANGES_REQUESTED** [review]

**Review by abhvsn — APPROVED** [review]
LGTM

**Comment by nayan-rafiq** [comment]
Waiting for the FE changes to be merged first.

**Comment by nayan-rafiq** [comment]
Closing this because this commit has been added to the branch: `epic/appsmith-design-system-v2` so that FE and BE changes are live together.

**Pull request closed without merging**